dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql

```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
 CREATE TABLE BANK USING column OPTIONS(partition_by 'BNK_ORG_ID', buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID ' ) AS (SELECT * FROM staging_BANK);
 
+DROP TABLE IF EXISTS BENEFIT_PACKAGE;
+DROP TABLE IF EXISTS staging_BENEFIT_PACKAGE;
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BENEFIT_PACKAGE
+    USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BENEFIT_PACKAGE.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```

Comment:
Parametrize the variables in sql script and pass them through bt or conf file at runtime.

Reply:
Done